ra/ra.go

```diff
@@ -1874,6 +1874,20 @@ func (ra *RegistrationAuthorityImpl) DeactivateAuthorization(ctx context.Context
 	return nil
 }
 
+// checkOrderNames validates that the RA's policy authority allows issuing for
+// each of the names in an order. If any of the names are unacceptable an
+// unauthorized error with suberrors for each rejected identifier is returned.
```

Comment:
I think the problem type will be either "malformed" or "rejectedIdentifier".

```suggestion
// error with suberrors for each rejected identifier is returned.
```

Reply:
Good call, fixed in https://github.com/letsencrypt/boulder/pull/4245 since this was merged.